src/Umbraco.Core/Services/UserService.cs

```diff
@@ -2103,6 +2104,40 @@ public async Task<Attempt<UserOperationStatus>> VerifyInviteAsync(Guid userKey,
         return changePasswordAttempt;
     }
 
+    public async Task<Attempt<PasswordChangedModel, UserOperationStatus>> ResetPasswordAsync(Guid performingUserKey, Guid userKey)
```

Comment:
It is customary to have the "performing user" be the last parameter in method signatures. 

Suggest refactoring this method and also `ResendInvitationAsync()`, which has the same parameter order.

Reply:
The user service seems the be the only one where putting it in front is done most often.. time to change a lot of method signatures...

Reply:
I would say putting it in the front is a more maintainable custom as it does not collide with params or optional parameters.

Reply:
Renamed all the params for clarity.